### ISSUE TITLE
improve EKM import key screen response

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#4335]
There are still some things left to adjust, but that will be for another issue:

 - following click of the setup button, there is no visual indication - no loader / spinner. We should make the button `gray` color with text `Loading..` and disable click events on it. Then the try/catch I just added should have a `finally` that re-enables the button
 - the EKM timeout is 30 seconds which is too long. Should make it 15 seconds for the EKM call

As it is now, user has to sit there and wait 30 seconds to see any UI change in some situations, even though I have fixed the issue. Only after that does the modal appear.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/issues/4334#issuecomment-1065320946_
